sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala

```diff
@@ -118,7 +118,45 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
           }
       """
     }.mkString("\n")
-    comparisons
+
+    /*
```

Comment:
@kiszk @ueshin
i kind of implemented a prove of concept (https://github.com/lw-lin/spark/commit/d0c11984e9aa84f68b9ebdd21317f629d759ae86) on how to extend `splitExpressions` to rewrite this PR
could you comment?